lib/vmpooler/vsphere_helper.rb

```diff
@@ -99,6 +99,14 @@ def find_pool(poolname)
       base
     end
 
+    def find_snapshot(vm, snapshotname)
+      if vm.snapshot
+        get_snapshot_list(vm.snapshot.rootSnapshotList, snapshotname)
+      else
+        raise "VM #{vm.name} has no snapshots to revert to"
```

Comment:
What about just returning an empty list instead of `raise`? Is not having a snapshot a reason to throw an error?